sonar-php-plugin/src/test/java/org/sonar/plugins/php/MockUtils.java

```diff
@@ -42,4 +46,8 @@ public static File getModuleBaseDir() {
     return new File("src/test/resources");
   }
 
+  public static <T extends Serializable> void assertMeasure(SensorContextTester context, String componentKey, org.sonar.api.measures.Metric<T> metric, T expected) {
```

Comment:
@ivandalbosco again this method has nothing to do with mocks. I would suggest to rename this class to remove `mock` from its name